marconi-sidechain/test/Spec/Marconi/Sidechain/CLIInputValidation.hs

```diff
@@ -0,0 +1,144 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+
+module Spec.Marconi.Sidechain.CLIInputValidation where
+
+import Control.Monad.IO.Class (liftIO)
+import Data.Functor ((<&>))
+import Hedgehog (Property, (===))
+import Hedgehog qualified as H
+import Hedgehog.Extras qualified as H
+import Spec.Marconi.Sidechain.Utils qualified as U
+import System.IO qualified as IO
+import System.Process qualified as IO
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.Hedgehog (testPropertyNamed)
+
+tests :: TestTree
+tests =
+  testGroup
+    "marconi-sidechain input validation error"
+    [ testGroup "check input validation when starting marconi-sidechain with invalid cli flags" $
+        invalidOptionalCliFlagsWithExpectedErrors
```

Comment:
Yeah, golden is probably best for these. Will adapt.

Reply:
Done in e68c13d3b608aa2196ca4c03df1a4e740818d3ca. Although, because the test previously used `Property` it meant we could leverage Hedgehog.Extras functionality to locate the executable under test using cabal, we now have that capability inside of marconi-sidechain for outside of the `Property` context, so no need to set an environment variable to specify its location.